RHDEVDOCS-4216: Add enable / disable quickstart docs (#60509)
openshift/openshift-docs: modules/disable-quickstarts-admin-console.adoc

// Module included in the following assemblies:
//
// * web_console/configuring-web-console.adoc

[id="disable-quickstarts-admin-console_{context}"]
= Disabling quick starts in the web console

You can use the *Administrator* perspective of the web console to disable one or more quick starts.

.Prerequisites

* You have cluster administrator permissions and are logged in to the web console.

.Procedure

. In the *Administrator* perspective, navigate to *Administation* -> *Cluster Settings*.

> . In the *Cluster Settings* page, click the *Configuration* tab.

Reviewer: ```suggestion
. On the *Cluster Settings* page, click the *Configuration* tab.
```
Per ISG, use "on" for web pages: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#page

> . In the *Configuration* page, click the *Console* configuration resource with the description *operator.openshift.io*.

Reviewer: ```suggestion
. Pn the *Configuration* page, click the *Console* configuration resource with the description *operator.openshift.io*.
```

+
image::cluster-settings-console.png[Image of the Configuration page showing the correct Console configuration resource to select]

> . In the *Action* drop-down list, click *Customize*. This takes you to the *Cluster configuration* page.

Reviewer: ```suggestion
. From the *Action* drop-down list, select *Customize*, which opens the *Cluster configuration* page.
```
- Per ISG use "from" and "select" when documenting actions for drop-down lists: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list
- Consider combining the sentences to show cause and effect more directly.

+
image::cluster-configuration-general-tab.png[Cluster configuration page open on the General tab]

> . In the *General* tab, under *Quick starts*, you can select items in either the *Enabled* or *Disabled* list, and move them from one list to the other by using the arrow buttons.

Reviewer: ```suggestion
. On the *General* tab, in the *Quick starts* section, select items in either the *Enabled* or *Disabled* list, and move them from one list to the other by using the arrow buttons.
```
- Per ISG, use "on" with tabs: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tab
- Replace "under". Per ISG, for accessibility reasons, we shouldn't refer user interface elements by using sensory words: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements

> ** To enable or disable a single quick start, click on the quick start, then use the single arrow buttons to move the quick start to the appropriate list.

Reviewer: ```suggestion
** To enable or disable a single quick start, click the quick start, then use the single arrow buttons to move the quick start to the appropriate list.
```
- Per ISG, don't use "on" with mouse actions like clicking: https://www.ibm.com/docs/en/ibm-style?topic=elements-mouse-buttons

> ** To enable or disable multiple quick starts at once, you can hold Ctrl and click the quick starts you want to move, before using the single arrow buttons to move the quick starts to the appropriate list.

Reviewer: ```suggestion
** To enable or disable multiple quick starts at once, press Ctrl and click the quick starts you want to move. Then, use the single arrow buttons to move the quick starts to the appropriate list.
```
- Because this is part of a procedure, consider using the imperative rather than "you can".
- Per ISG, use "press" with keyboard function keys like Ctrl: https://www.ibm.com/docs/en/ibm-style?topic=elements-keyboard-keys#verbs-with-keyboard-keys
- Consider splitting this one long sentence into two.
